python/pyspark/sql/functions.py

```diff
@@ -809,6 +809,48 @@ def ntile(n):
     return Column(sc._jvm.functions.ntile(int(n)))
 
 
+@since(2.3)
+def unboundedPreceding():
+    """
+    Window function: returns the special frame boundary that represents the first row
+    in the window partition.
+
+    >>> df = spark.createDataFrame([(5,)])
+    >>> df.select(unboundedPreceding()).columns[0]
+    'UNBOUNDED PRECEDING'
```

Comment:
Shall we just put these tests as separate tests in `tests.py`? I don't think this is quite useful as an example in doc to be honest ..

Reply:
I guess I will just remove these tests since we already have a doc test in window.py.